packages/ripple-nuxt-tide/modules/preview/lib/helpers.js

```diff
@@ -59,7 +59,8 @@ async function getPageByPreviewLink ($tide, path, site, section, headersConfig)
 async function tideShare (context, headersConfig) {
   const siteId = (context.store.state.tide && context.store.state.tide.siteId) ? context.store.state.tide.siteId : null
   const section = (context.route.query && context.route.query.section) ? context.route.query.section : null
-  return getPageByShareLink(context.app.$tide, context.route.path, siteId, section, headersConfig)
+  const path = context.route.path[0] === '/' ? context.route.path.substring(1) : context.route.path
```

Comment:
Hi @alan-cole not sure why this... can you explain it please?

Reply:
@tim-yao 

It's to avoid a double slash in the axios path. It's not necessary as the URL will still resolve, but it felt more correct to be passing well formed URLs than leaving it to axios to fix.
I'm happy to remove if you'd prefer the keep the code lean.

```js
const path = '/share_link/952b5923-7dce-4315-9b6a-2ead8e381be1/13358'

$tide.getByURL(path)

const apiPrefix = '/api/v1/'

function getByURL (url, headersConfig = {}) {
  return axios.$get(`${apiPrefix}${url}`, this._axiosConfig(headersConfig))
}

// axios request becomes:

// /api/v1//share_link/952b5923-7dce-4315-9b6a-2ead8e381be1/13358
//        ^
// This code removes this double //
```

Reply:
Thanks, I am happy to keep your code. @alan-cole